ecl/hthor/hthor.ipp

```diff
@@ -3239,6 +3239,10 @@ protected:
     void onLimitExceeded();
 };
 
+// improvement: override constructor to store setHttpIdHeaderNames, override CTXLOG & logOperatorExceptionVA to log LogMsgJobInfo
+class CHThorContextLogger : public CThorBaseContextLogger
+{
+};
```

Comment:
With a rename of the base to CStatsContextLogger (+ passing in mapping + job), 
I think this can be deleted in favour of using CStatsContextLogger.
Thor's derivation can remain for it's specialization if need be.